helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -62,10 +62,10 @@ splunkPlatform:
   #       The file will be stored as a secret in kubernetes.
   caFile: ""
 
-  # Options to disable particular telemetry data types that will be sent to
-  # Splunk Platform.
+  # Options to disable or enable particular telemetry data types that will be sent to
+  # Splunk Platform. Only logs collection is enabled by default.
   logsEnabled: true
-  metricsEnabled: true
+  metricsEnabled: false
```

Comment:
Why are we disabling metrics by default?

Reply:
because user must create metric type index and provide that name to `splunkPlatform.metrics_index`

Reply:
Yeah, we want to make sure that chart is working by default just with `token` and `endpoint` as stated in the docs

Reply:
```suggestion
  # If you enable metrics collection, make sure that `metrics_index` is provided as well. 
  metricsEnabled: false
```